app-modules/authorization/src/Models/Role.php

```diff
@@ -81,7 +80,7 @@ public function scopeWeb(Builder $query): void
     public function scopeSuperAdmin(Builder $query): void
     {
         $query
-            ->where('name', Authenticatable::SUPER_ADMIN_ROLE)
+            ->where('name', 'SaaS Global Admin')
```

Comment:
I think this should still be and use the const set by `Authenticatable`. I believe that is what we do in Advising App as well.

Reply:
Updated